tests/test_genetic_maps.py

```diff
@@ -64,7 +64,7 @@ def __init__(self):
 
 # TODO add some parameters here to check different compression options,
 # number of chromosomes etc.
-def get_genetic_map_tarball():
+def get_genetic_map_tarball(custom_file_f=None, filter=None):
```

Comment:
Cunning! A quick comment here noting what the parameters are for would help the reader though.